packaging/cmake/Modules/Packaging.cmake

```diff
@@ -93,10 +93,10 @@ endif()
 list(JOIN _main_deps ", " CPACK_DEBIAN_NETDATA_PACKAGE_DEPENDS)
 
 set(CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA
-	  "${CMAKE_SOURCE_DIR}/packaging/cmake/control/netdata/conffiles;"
-	  "${CMAKE_SOURCE_DIR}/packaging/cmake/control/netdata/preinst"
-	  "${CMAKE_SOURCE_DIR}/packaging/cmake/control/netdata/postinst"
-	  "${CMAKE_SOURCE_DIR}/packaging/cmake/control/netdata/postrm")
+	  "${PKG_FILES}/deb/netdata/conffiles;"
```

Comment:
@Ferroin You  forgot to rename these to PKG_FILES_PATH.

Reply:
Should be resolved now (as well as the merge conflict).